src/modules/fix_widescreen.rs

```diff
@@ -0,0 +1,70 @@
+//! `bxt_fix_widescreen_fov`
+
+use super::Module;
+use crate::hooks::engine;
+use crate::modules::cvars::CVar;
+use crate::utils::*;
+
+pub struct FixWidescreen;
+impl Module for FixWidescreen {
+    fn name(&self) -> &'static str {
+        "bxt_fix_widescreen_fov"
+    }
+
+    fn description(&self) -> &'static str {
+        "Fixes widescreen vertical field-of-view."
+    }
+
+    fn is_enabled(&self, marker: MainThreadMarker) -> bool {
+        engine::R_SetFrustum.is_set(marker) && engine::scr_fov_value.is_set(marker)
+    }
+
+    fn cvars(&self) -> &'static [&'static CVar] {
+        static CVARS: &[&CVar] = &[&BXT_FIX_WIDESCREEN_FOV];
+        CVARS
+    }
+}
+
+static BXT_FIX_WIDESCREEN_FOV: CVar = CVar::new(
+    b"bxt_fix_widescreen_fov\0",
+    b"0\0",
+    "\
+Fixes reduction in vertical field-of-view of widescreen. Compatible with `bxt_force_fov`",
+);
+
+// My guess is that every render cycle, `scr_fov_value` is reset to some values.
```

Comment:
It's set in the client DLL in HUD_something which runs every frame but not during loads.